tests/src/rules/dynamic-import-chunkname.js

```diff
@@ -53,10 +53,34 @@ ruleTester.run('dynamic-import-chunkname', rule, {
         'someModule'
       )`,
       options: pickyCommentOptions,
-      errors: [{
-        message: pickyCommentFormatError,
-        type: 'CallExpression',
-      }],
```

Comment:
why are these tests removed? (removing tests is a red flag that it might be a breaking change)

Reply:
> why are these tests removed? (removing tests is a red flag that it might be a breaking change)

I removed unused `errors` property, test itself is still in place. There are multiple tests which are expected to be valid, but they contain `errors` field, which is AFAIK not used, because test passes. Probably someone was just copy/pasting tests and forgot to remove it.

Reply:
Ahhh thanks :-) good catch!